### PR TITLE
auto OCP-21772 check user starter guide on projects

### PR DESCRIPTION
**Pull request by yapei**
@yanpzhan  @SSshahan   Please help review, thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.2/status.xyaml [hunk 1]
```diff
@@ -27,3 +27,13 @@ check_dim_resource_list:
...
+check_user_starter_guide_message:
+  elements:
+  - selector:
+      visible_text: !ruby/regexp /Welcome to OpenShift/i
+  - selector:
+      visible_text: !ruby/regexp /Create a project for your application/i
+check_get_started_message_when_no_resources:
+  elements:
+  - selector:
+      visible_text: !ruby/regexp /No Workloads Found/i
```
Comment (SSshahan) [inline lib/rules/web/admin_console/4.2/status.xyaml]: Curiously, why we didn't use the existing `check_page_contains` rule?

Comment (yapei) [reply lib/rules/web/admin_console/4.2/status.xyaml]: For current 4.2 implementation, only `No Workloads Found` message and can be replaced with 
```
check_get_started_message_when_no_resources:
  params:
    content: No Workloads Found
  action: check_page_match
```

Comment (yapei) [reply lib/rules/web/admin_console/4.2/status.xyaml]: Is this what you mean?

**Comment by SSshahan** [comment]
Thanks for updates. LGTM

**Pull request merged**